02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,73 @@
+  <h1>Claire Plunket</h1>
+    <p>1234 S. Congress Avenue <br>
+    Austin, Texas<br>
+    214-555-5555<br>
+    <a href="mailto:claire@email.com">Email Me</a><br>
+    <a href="https://www.instagram.com/claireplunket/">Follow me on Instagram</a>
+    </p>
+  <h3><em>About Me</em></h3>
+    <p>I was born and raised in Dallas, Texas. After graduating from the University of Virginia with degrees in Mathematics and Spanish,
+      I moved to Arlington, Virginia to begin my career in Client Services at a tech company. After a couple years, I moved with the company
+      down to Austin to lead the Project Management team of their newly acquired Mobile Application Development company. I am now a Digital
+      Project Manager at a Digital Agency with a special interest in website redesign projects.
+    </p>
+    <table class="center">
+      <thead>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <img src="https://scontent-dft4-2.cdninstagram.com/t51.2885-15/e15/10986004_435900576563366_1287462720_n.jpg?ig_cache_key=OTI3MDkxODE4OTMzNTU1MTE2.2" alt="Skiing"/>
+          </td>
+          <td>
+            <img src="https://scontent-dft4-2.cdninstagram.com/t51.2885-15/e35/11856754_173960306268799_61539940_n.jpg?ig_cache_key=MTA2MTg0MDg5NDg5NTA5OTgzMA%3D%3D.2"/>
+          </td>
+          <td>
+            <img src="https://scontent-dft4-2.cdninstagram.com/t51.2885-15/e15/1515629_403274086473425_1836824376_n.jpg?ig_cache_key=NjE2ODA5MzY2ODc0Nzg3MjI3.2"/>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <img src="https://scontent-dft4-2.cdninstagram.com/t51.2885-15/s640x640/sh0.08/e35/12292733_1516163668682392_1877789586_n.jpg?ig_cache_key=MTEyNTQyNDcxNDA4MDM1NzQyOA%3D%3D.2"/>
+          </td>
+          <td>
+            <h4>Things I Love Doing</h4>
+            <ul>
+              <li>Skiing</li>
+              <li>Hiking</li>
+              <li>Traveling</li>
+              <li>Dancing</li>
+              <li>Running</li>
+              <li>Festivaling</li>
+              <li>Beaching</li>
+              <li>Board Gaming</li>
+            </ul>
+          </td>
+          <td>
+            <img src="https://scontent-dft4-2.cdninstagram.com/t51.2885-15/e15/11254506_447182862120342_104637431_n.jpg?ig_cache_key=MzkzNzU5NDE4ODk1MDc2ODk3.2"/>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <img src="https://scontent-dft4-2.cdninstagram.com/t51.2885-15/e15/10254065_1469614773327543_1771233667_n.jpg?ig_cache_key=ODI1MzU3Mjk1NDQ5NTY0MzQ3.2"/>
+          </td>
+          <td>
+            <img src="https://scontent-dft4-2.cdninstagram.com/l/t51.2885-15/e35/13712138_534566276731840_1456691937_n.jpg?ig_cache_key=MTI5Mzk0MzMwMDI4NjI2ODU3Ng%3D%3D.2"/>
+          </td>
+          <td>
+            <img src="https://scontent-dft4-2.xx.fbcdn.net/t31.0-8/13668739_10208180628487020_8075204476830363759_o.jpg"/>
+          </td>
+        </tr>
+      </tbody>
+    </table>
 
 </body>
 </html>
```

Comment:
Fantastic! Great design and layout. Creative use of a table. I'm glad you made it back to Texas, although Virginia is also beautiful.